code/datums/components/jetpack.dm

```diff
@@ -17,6 +17,12 @@
 	var/datum/effect_system/trail_follow/trail
 	/// The typepath to instansiate our trail as, when we need it
 	var/effect_type
+	/// Drift force applied each movement tick
+	var/drift_force
+	/// Force that applied when stabiliziation is active and the player isn't moving in the same direction as the jetpack
+	var/stabilization_force
+	/// Our current user
+	var/mob/user
```

Comment:
why?

Reply:
stabilization is ran in process() so it needs to keep its user somewhere